src/App/Pages/Vault/GroupingsPage/GroupingsPage.xaml.cs

```diff
@@ -222,6 +223,10 @@ private async void RowSelected(object sender, SelectionChangedEventArgs e)
             {
                 await _vm.SelectTrashAsync();
             }
+            else if (item.IsTotpCode)
+            {
+                await _vm.SelectTotpCodesAsync();
+            }
```

Comment:
Given this is called from `async void RowSelected`, if an exception is raised it will crash the app. IMO, this should be refactored to have a command that takes the `item` and in the VM calls the right method.
For now, at least a try...catch should be added to the whole `RowSelected` to avoid potential crashes and to log the exception